src/frontend/src/btc/components/fee/UtxosFeeContext.svelte

```diff
@@ -20,7 +21,7 @@
 			return;
 		}
 
-		if (isNullish(networkId) || isNullish(amount) || amount === 0) {
+		if (amountError || isNullish(networkId) || isNullish(amount) || Number(amount) === 0) {
```

Comment:
`Number(amount)` is related to the PR?

Reply:
Not really, I didn't notice this change slipped through. Will create a separate PR for that.